Detector/DetFCChhECalInclined/src/ECalBarrelInclined_geo.cpp

```diff
@@ -185,6 +185,21 @@ static dd4hep::detail::Ref_t createECalBarrelInclined(dd4hep::Detector& aLcdd,
        << " rotation angle = " << angle << endmsg;
   uint numPlanes =
       round(M_PI / asin((passiveThickness + activeThickness + readoutThickness) / (2. * caloDim.rmin() * cos(angle))));
+
```

Comment:
One last thing: can you add a comment here stating why this check is there and why numPlates has to be defined in the xml?

Reply:
Done in latest commit.